qa/standalone/test_ceph_daemon.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash -ex
+
+[ -z "$SUDO" ] && SUDO=sudo
+[ -x ../src/ceph-daemon ] && CEPH_DAEMON=../src/ceph-daemon
+[ -x ./ceph-daemon ] && CEPH_DAEMON=.ceph-daemon
+which ceph-daemon && CEPH_DAEMON=$(which ceph-daemon)
+
+FSID='00000000-0000-0000-0000-0000deadbeef'
+IMAGE='ceph/daemon-base:latest-master'
```

Comment:
Can we make use of the new images on quay.io

Reply:
which image?  right now quay.io only gets the wip- branches for some annoying ceph-build reason